docs/pages/router/advanced/custom-tabs.mdx

```diff
@@ -6,6 +6,7 @@ sidebar_title: Custom tabs
 
 import { Collapsible } from '~/ui/components/Collapsible';
 import { FileTree } from '~/ui/components/FileTree';
+import { Tabs, Tab } from '~/ui/components/Tabs';
```

Comment:
```suggestion
```

Since this component is not used on this page anymore, let's remove it as per the CI warning.